src/shaders/encode_attribute.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const util = require('../util/util');
+
+/**
+ * Packs two numbers, interpreted as 8-bit unsigned integers, into a single
+ * float.  Unpack them in the shader using the `unpack_float()` function,
+ * defined in _prelude.vertex.glsl
+ *
+ * @private
+ */
+exports.packUint8ToFloat = function pack(a, b) {
```

Comment:
Oh wait, no -- I think we'll need this, it's just still TBD: we still need to pack `a_data` together with `a_texture_pos` so that there's no net increase in number of attributes, and this helper will be used for that.